apps/foundation-demo/src/app/design/atoms/atoms.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+
+import { ButtonComponent } from './button/button.component';
+
+@NgModule({
+    imports: [
+      CommonModule,
+      FormsModule
+    ],
+    declarations: [
+      ButtonComponent
+    ],
+    exports: [
+      ButtonComponent
```

Comment:
The error is coming from here. You're declaring it here and in the button.module. You should import `ButtonModule` here instead.

Reply:
Let's remove this module entirely. Nothing should be importing this whole thing. Components should import as needed to reduce bundle size.